F/FastANI/buildtarballs.jl

```diff
@@ -0,0 +1,50 @@
+using BinaryBuilder, Pkg
+
+name = "FastANI"
+version = v"1.33.0"
+sources = [
+    GitSource("https://github.com/ParBLiSS/FastANI.git", "6531da0957344d2a56e3eeb9bb83fea02cc4abe0"),
+]
+
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/FastANI
+
+update_configure_scripts --reconf
+autoupdate
+
+./bootstrap.sh
+./configure \
+    --prefix=$prefix \
+    --build=${MACHTYPE} \
+    --host=${target} \
+    --with-gsl=${destdir} \
+    --includedir=${includedir} \
+    --libdir=${libdir}
+    
+if [[ $target == *-apple-* ]]; then
+    MACFLAGS="-mmacosx-version-min=10.7 -stdlib=libc++ -Xpreprocessor -lomp"
```

Comment:
I don't understand why this'd be needed.  We use `-mmacosx-version-min=10.10` by default for x86_64, and 11.0 for aarch64, setting 10.7 doesn't make much sense.  Also, setting `-Xpreprocessor -lomp` is likely useless, `-fopenmp` should be enough.